src/cdk/a11y/focus-trap/configurable-focus-trap-config.ts

```diff
@@ -6,6 +6,6 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
- export class ConfigurableFocusTrapConfig<D = any> {
+ export class ConfigurableFocusTrapConfig {
   defer: boolean = false;
```

Comment:
We should have docs on the class and the `defer` property since it's not immediately obvious what deferring means in this case.

Reply:
Was this just a request for adding comments or API docs? It looks like this Class isn't in the [API docs](https://material.angular.io/cdk/a11y/api) since it wasn't added to the public API in
https://github.com/angular/components/blob/c7edf03a023220a7a230902aa5194e65435e89ae/src/cdk/a11y/public-api.ts#L12-L16